### PR TITLE
Consolidate reviews

### PR DESCRIPTION
**Pull request by aaronpeikert**
The rendered response letter look like this: 
[reviews.pdf](https://github.com/user-attachments/files/16871826/reviews.pdf)
The diff link will only work when this is merged into main. In the meantime: https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0..consolodate-reviews#diff-a9a4aad3fa8c9c10c5404b632bc3a01a25d2d8430eb932bc35c76769963e4b70

### CODE REVIEW

File: reviews.Rmd [hunk 1]
```diff
@@ -3,23 +3,26 @@ title          : "Why does preregistration increase the persuasiveness of eviden
...
 
 thank you very much for taking the time to consider our manuscript for publication at _`r rmarkdown::metadata$journal`_.
 In the following we address your and each reviewers'/editors' concerns point-by-point.
-For Daniel Laken's review we took the liberty to additionally include a high level summary, since we found it, at times, hard to follow the different interwoven discussions.
 
 All changes of the manuscript since this review can be viewed on GitHub:
 
 [https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main](https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main#diff-a9a4aad3fa8c9c10c5404b632bc3a01a25d2d8430eb932bc35c76769963e4b70)
 
+For Daniel Laken's review we took the liberty to additionally include a high level summary, since we found it, at times, hard to follow the different interwoven discussions.
```
Comment (brandmaier) [inline reviews.Rmd:21]: make it less judgemental?

```suggestion
Concerning Daniel Laken's review, we took the liberty to provide a high level summary of his main points because they are repeated across different subdiscussions throughout his letter.
```

File: reviews.Rmd [hunk 2]
```diff
@@ -3,23 +3,26 @@ title          : "Why does preregistration increase the persuasiveness of eviden
...
+We took special care to change the manuscript in response to his summary of suggestions (RC \@ref(pt:change-start) to RC \@ref(pt:change-end)).
+
+Due to a technical issue, the citations in the manuscript excerpts in this letter do not work correctly (they are correct in the manuscript, however).
```
Comment (brandmaier) [inline reviews.Rmd:24]: is there no way to fix this? What is the technical issue? A bug?

File: reviews.Rmd [hunk 3]
```diff
@@ -3,23 +3,26 @@ title          : "Why does preregistration increase the persuasiveness of eviden
...
+
 # Daniel Lakens
```
Comment (brandmaier) [inline reviews.Rmd:26]:
```suggestion
# Reviewer 1: Daniel Lakens
```

File: reviews.Rmd [hunk 4]
```diff
@@ -3,23 +3,26 @@ title          : "Why does preregistration increase the persuasiveness of eviden
...
```
Comment (brandmaier) [inline reviews.Rmd:27]:
```suggestion
We begin with a summarizing our understanding of Daniel Lakens philosophical position, his main points of criticisim and our general take on how to address these issues. Thereafter, we will provide point-by-point responses in the usual style of response letters.
```

File: reviews.Rmd [hunk 5]
```diff
@@ -3,23 +3,26 @@ title          : "Why does preregistration increase the persuasiveness of eviden
...
 
 Lakens and our own positions on preregistration built upon two long standing traditions in philosophy of science, namely inductionist and anti-inductionist, that differ fundamentally in their core assumptions.
```
Comment (brandmaier) [inline reviews.Rmd:28]:
```suggestion
Lakens and our own positions on preregistration fundamentally differ. Yet, they each build upon two long standing traditions in philosophy of science, namely _inductionist_ and _anti-inductionist_, which differ fundamentally in their core assumptions. We believe that a lot of seemingly critical points raised by Daniel Lakens can simply be traced back to differences in these philosophical stances and thus cannot be resolved in the various sub points he raised. 
```

File: reviews.Rmd [hunk 6]
```diff
@@ -57,7 +61,8 @@ Type I and Type II error rates:
 * Our Response: The Bayesian rational considers theoretical risk and detectability simultaneously a priori. From a severity perspective, either Type I or Type II, never both, is considered post hoc, depending on the test results and if the research hypothesis is H1 or H2.
```
Comment (brandmaier) [inline reviews.Rmd:61]: Please clarify this sentence "From a severity perspective,..." What is H1 or H2 here? What is posthoc here?

**Comment by brandmaier** [comment]
Please see my PR for additional suggestions, which I couldn't add as comments.

**Review by brandmaier — CHANGES_REQUESTED** [review]
Approve conditional on additional suggestions in PR

File: reviews.Rmd (hunk 6 above)
Comment (aaronpeikert) [reply reviews.Rmd:61]:
```suggestion
* Our Response: The Bayesian rational considers theoretical risk and detectability simultaneously a priori. From a severity perspective, either Type I or Type II, never both, is considered post hoc (i.e., after a test result was obtained from data) depending on the test results and if the research hypothesis is H0 or H1. 
```
